packages/pwa-kit-react-sdk/src/utils/url.js

```diff
@@ -12,6 +12,8 @@
  * initialized using the `_createApp` method (This happens in your /app/ssr.js file).
  *
  * @function
+ * @deprecated since version 3.8.0 - use `useOrigin()` instead.
```

Comment:
Is it ok if we don't specify the version here? Because the version this feature will get released in may not be what we expect.

Reply:
it said since 3.8.0 which is the to-be-released for this package. anything after that still applies.

Reply:
If this feature is actually to be released _later_ in v3.9, I'm not confident we'll remember to update this comment.